README.md

```diff
@@ -302,6 +311,18 @@ Useful fun activities that can be done at the meetup/events/activities.
 - [TypeRacer](https://play.typeracer.com/) - TypeRacer is a fun way to engage people during a meetup while improving their typing skills.
 </details>
 
+<details>
+  <summary>
+    👨🏻‍💻 OpenSource
+  </summary>
+
+<br />
+
+OpenSource is for everyone.
```

Comment:
I'm not sure this category qualifies as a resource since Hactoberfest is an event/ challenge while FOSSASIA is an organization.

Would you mind removing this section?